ops/terraform/services/base/values/test.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# COMMON CONFIGURATION
+/bfd/test/common/nonsensitive/enterprise_tools_security_group: bfd-test-enterprise-tools
+/bfd/test/common/nonsensitive/env_name_std: test
+/bfd/test/common/nonsensitive/key_pair: bfd-test
+/bfd/test/common/nonsensitive/kms_key_alias: alias/bfd-test-cmk
+/bfd/test/common/nonsensitive/management_security_group: bfd-test-remote-management
+/bfd/test/common/nonsensitive/new_relic_metric_host:  https://gov-metric-api.newrelic.com
+/bfd/test/common/nonsensitive/new_relic_metric_path: /metric/v1
+/bfd/test/common/nonsensitive/rds_cluster_identifier: bfd-test-aurora-cluster
+/bfd/test/common/nonsensitive/rds_instance_class: db.r5.12xlarge
+/bfd/test/common/nonsensitive/rds_instance_count: 2
```

Comment:
not asking for a change here but is this what we have converged on as the official value and accepting the difference with prod?

Reply:
Thanks, @christrotteradhoc. No–this should match prod, but we'll be sure to address this in [BFD-2006](https://jira.cms.gov/browse/BFD-2006).